src/backend/crew_ai/optimization/conflict_detection.py

```
@@ -0,0 +1,307 @@
+                        feedback_loop.write_trigger_event,
+                        trigger_type=trigger_type,
+                        workflow_id=workflow_id,
+                        domain=domain, url=url,
+                        feedback_text=feedback_text,
+                        active_hint_ids=[], flagged_hint_ids=[],
+                        actually_flagged_hint_ids=[],
+                        reason=None, llm_model=None,
+                        input_tokens=0, output_tokens=0, llm_latency_ms=0,
+                        status="no_active_hints", error_message=None,
+                    )
+                except Exception as tel_err:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)